File: KCRI-ePR/wwwroot/js/staticData.js
let staticData = new function () {

    let companyItem = [
        { ItemValue: 1, ItemText: "KCRI" },
        { ItemValue: 2, ItemText: "CHLL" }
    ];

    // Add a method to get the companyItem data
    this.getCompanyItems = function () {
        return companyItem;
    }

    let devisionItem = [
        { ItemValue: 1, ItemText: "CGs" },
        { ItemValue: 2, ItemText: "MSG" }
    ];

    // Add a method to get the companyItem data
    this.getDevisionItems = function () {
        return devisionItem;
    }

}

File: KCRI-ePR/Data/StaticData.cs
using Microsoft.AspNetCore.Mvc.Rendering;

namespace KCRI_ePR.Data
{
    public static class StaticData
    {
        public static List<SelectListItem> CompanyList => new List<SelectListItem>
        {
            new SelectListItem { Text = "KCRI", Value = "1" },
            new SelectListItem { Text = "CHLL", Value = "2" }
        };

        public static List<SelectListItem> DivisionList => new List<SelectListItem>
        {
            new SelectListItem { Text = "CGs", Value = "1" },
            new SelectListItem { Text = "MSG", Value = "2" }
        };
    }
}
